app/models/rental.rb

```diff
@@ -0,0 +1,26 @@
+class Rental < ApplicationRecord
+  belongs_to :customer
+  belongs_to :movie
+
+  validates :customer_id, presence: true
+  validates :movie_id, presence: true
+
+
+  def rent_movie(customer, movie)
+    checked_out = customer.movies_checked_out_count
+    customer.update(movies_checked_out_count: checked_out +1)
```

Comment:
Minor syntax thing: Ruby linters complain if in Ruby you don't put a space before `1` (it would prefer `+ 1`). Same with the minus sign below on line 20